# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1318031087 25200
#      Fri Oct 07 16:44:47 2011 -0700
# Node ID c479e3d0311fd851daaadeba3eccb4e9d150c31b
# Parent  1a6537dbce7b95b9217ccb83fd35e70648f48ce0
shift.m: Fix typo in %!error tests

* shift.m: Fix typo in %!error tests

diff --git a/scripts/general/shift.m b/scripts/general/shift.m
--- a/scripts/general/shift.m
+++ b/scripts/general/shift.m
@@ -87,12 +87,12 @@ endfunction
 %! assert(shift (m, 1), [c; a; b]);
 %! assert(shift (m, -2), [c; a; b]);
 
 %% Test input validation
 %!error shift ()
 %!error shift (1, 2, 3, 4)
 %!error shift ([], 1)
 %!error shift (ones(2), ones(2))
-%!error kurtosis (ones(2), 1.5)
-%!error kurtosis (ones(2), 0)
-%!error kurtosis (ones(2), 3)
+%!error shift (ones(2), 1.5)
+%!error shift (ones(2), 0)
+%!error shift (ones(2), 3)
 
